plugins/mcp-actions-backend/README.md

```diff
@@ -103,6 +103,29 @@ node -p 'require("crypto").randomBytes(24).toString("base64")'
 
 Set the `MCP_TOKEN` environment variable with this token, and configure your MCP client to use it in the [Authorization header](#configuring-mcp-clients)
 
+#### Experimental: Dynamic Client Registration
+
+> This is highly experimental, proceed with caution.
```

Comment:
```suggestion
> [!CAUTION]
> This is highly experimental, proceed with caution.
```

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts